osutil/disks/disks.go

```diff
@@ -187,3 +187,25 @@ func (e PartitionNotFoundError) Error() string {
 var (
 	_ = error(PartitionNotFoundError{})
 )
+
+var deviceMapperHandlers = map[string]func(dmUUID, dmName []byte) (dev, devType string, ok bool){}
+
+// RegisterDeviceMapperHandler takes a callback function which is used when the
+// disks package through some of it's various methods to locate/create a disk
+// needs to trace a device mapper node back to the original device node location
+// such as /dev/mapper/something -> /dev/sda1 -> /dev/sda. The parameters the
+// callback is provided are the device mapper UUID and name parameters from the
+// kernel, and if and only if the device mapper handler matches this device
+// mapper node, the callback should return the source device node for the mapper
+// device, the device type option (either "name" or "path") of the device
```

Comment:
device type sounds confusing in this context.  it's more of a reference kind or some similar name?

Reply:
yeah it's kinda an implementation detail, but as mentioned since we only really need name for now, we can probably just drop that parameter entirely and only use name for everything now and some day if we need to use path we can figure it out then

Reply:
as mentioned, removed in https://github.com/snapcore/snapd/pull/11035